enterprise/server/remote_execution/commandutil/commandutil.go

```diff
@@ -215,6 +282,126 @@ func ChildPids(pid int) ([]int, error) {
 	return out, nil
 }
 
+// TreeStats records stats across the lifecycle of a process tree, accounting
```

Comment:
it feels like there is enough code here that maybe it warrants moving to its own package. wdyt?

Something like util/stats?